src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,113 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private static User user2 = new User();
+    private static User user3 = new User();
+    private static RegistrationServiceImpl registrationService;
+    private User defaultUser;
+
+    @BeforeAll
+    static void beforeAll() {
+        registrationService = new RegistrationServiceImpl();
+        String loginBruce = "Bruce";
+        int validUser2Age = 100;
+        String reverseGoodPassword = "654321";
+        user2.setLogin(loginBruce);
+        user2.setAge(validUser2Age);
+        user2.setPassword(reverseGoodPassword);
+        String loginAmy = "Amy";
+        int validEighteenAge = 18;
+        String goodPassword = "parol2";
+        user3.setLogin(loginAmy);
+        user3.setAge(validEighteenAge);
+        user3.setPassword(goodPassword);
+    }
+
+    @BeforeEach
+    void beforeEach() {
+        int validDefaultAge = 18;
+        String loginDefault = "Kate";
+        defaultUser = new User();
+        defaultUser.setLogin(loginDefault);
+        defaultUser.setAge(validDefaultAge);
+        String goodPassword = "123456";
+        defaultUser.setPassword(goodPassword);
+    }
+
+    @Test
+    void addUser_SameLogin_NotOk() {
+        registrationService.register(defaultUser);
+        assertThrows(RuntimeException.class, () -> registrationService.register(defaultUser));
+    }
+
+    @Test
+    void addUser_NullLogin_NotOk() {
```

Comment:
```suggestion
    void register_nullLogin_NotOk() {
```

fix naming for all tests